types/types.go

```diff
@@ -363,11 +370,22 @@ func (reports SpecReports) CountWithState(states SpecState) int {
 	return n
 }
 
-//CountWithState returns the number of SpecReports that passed after multiple attempts
+// CountOfFlakedSpecs returns the number of SpecReports that failed after multiple attempts
 func (reports SpecReports) CountOfFlakedSpecs() int {
 	n := 0
 	for i := range reports {
-		if reports[i].State.Is(SpecStatePassed) && reports[i].NumAttempts > 1 {
+		if reports[i].IsFlakeAttempts && reports[i].State.Is(SpecStatePassed) && reports[i].NumAttempts > 1 {
+			n += 1
+		}
+	}
+	return n
+}
+
+// CountOfRepeatedSpecs returns the number of SpecReports that passed after multiple attempts
```

Comment:
I don't think we need this - there's no need to tell the user that a spec was repeated if they asked for it.  Whereas we _do_ want to tell the user "hey, this suite passed but only because we retried some flaky specs". (hence `CountOfFlakedSpecs()`

Reply:
Answered in comment discussion:

> Based on this feedback, I'll change the `CountOfRepeatedSpecs` to only populate the `Repeated` report field when the test with the `MustPassRepeatedly` decorator fails. This was the user can understand whether a test failure is associated with a `MustPassRepeatedly` decorator or not.